grakn-graph/src/test/java/ai/grakn/generator/GraknGraphs.java

```diff
@@ -106,9 +107,12 @@ public GraknGraph generate() {
         boolean shouldOpen = open != null ? open : random.nextBoolean();
 
         if (shouldOpen) {
-            graph.open();
         } else {
-            graph.close();
+            try {
+                graph.close();
+            } catch (GraknValidationException e) {
+                throw new RuntimeException("Generater opened invalid graph somehow", e);
+            }
```

Comment:
Should invert the condition instead of an empty `if`. Also the `try...catch` is unnecessary now that `GraknValidationException` is runtime.